SettingsPanel.lua

```diff
@@ -927,6 +927,15 @@ function addon.settings:CreateAceOptionsPanel()
                                                           "OUTLINE")
                         end
                     },
+                    resetArrowPosition = {
+                        order = 5.4,
+                        name = L("Reset Arrow Position"), -- TODO locale
+                        type = "execute",
+                        width = "small",
```

Comment:
```suggestion
                        width = optionsWidth,
```

to keep it consistent with the rest